recipe/meta.yaml

```diff
@@ -433,9 +435,9 @@ outputs:
         - ls -A1 ${PREFIX}/bin/*         # [unix]
         - ls -A1 ${PREFIX}/include/*     # [unix]
         - ls -A1 ${PREFIX}/lib/*         # [unix]
-        - test -f ${PREFIX}/include/sycl/CL/sycl.hpp  # [unix]
         - dir %PREFIX%\Library\bin\*  # [win]
         - dir %PREFIX%\Library\lib\*  # [win]
+        # Appears to return a nonzero exit code
```

Comment:
not sure that i do understand context of this comment, @AndresGuzman-Ballen  could you please elaborate?